data/Nikhil202003.json

```
@@ -0,0 +1,36 @@
+{
+	"name": "P.Nikhil ",
+	"type": "personal",
+	"bio": "Open Source",
+	"tags": [
+		"Open source",
+		"java"
+	],
+	"social": [
+
+		{
+			"icon": "FaLinkedin",
+			"url": "https://www.linkedin.com/in/P Nikhil"
+		},
+		{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/ Pnikhil003"
```

There should be no spaces in the URLs. So please remove the extra spaces from URLs. 

Also, update it with your LinkedIn URL instead. 
```suggestion
			"icon": "FaLinkedin",
			"url": "https://www.linkedin.com/in/PNikhil"
		},
		{
			"icon": "FaTwitter",
			"url": "https://twitter.com/Pnikhil003"
```